modules/ocf_irc/manifests/xmpp.pp

```diff
@@ -1,19 +1,30 @@
 class ocf_irc::xmpp {
   # When upgrading to buster, these can be installed from the regular buster
   # repo
-  ocf::repackage {
-    # lua-dbi-mysql is needed to connect to MySQL
-    'lua-dbi-common':
-      backport_on => ['stretch'];
+  if $::lsbdistcodename != 'buster' {
```

Comment:
Same for all these, I don't think they should be actually needed since `ocf::repackage` should just install the package normally on dists not in the `backport_on` list.